pkg/obs/nodeserver.go

```diff
@@ -93,22 +93,17 @@ func (ns *nodeServer) NodePublishVolume(_ context.Context, req *csi.NodePublishV
 		return nil, status.Errorf(codes.Internal, "Failed to make dir: %s, error: %v", targetPath, err)
 	}
 
-	credentialFile := fmt.Sprintf("%s/%s", credentialDir, uuid.New().String())
+	credentialFile := filepath.Join(credentialDir, credentialFileName)
```

Comment:
The credential file is randomly generated each time to avoid conflicts.

Reply:
fix